packages/plugin-eslint/package.json

```diff
@@ -3,8 +3,16 @@
   "version": "0.1.0",
   "dependencies": {
     "@code-pushup/utils": "*",
+    "@code-pushup/models": "*",
     "eslint": "~8.46.0",
-    "zod": "^3.22.4",
-    "@code-pushup/models": "*"
+    "zod": "^3.22.4"
+  },
+  "peerDependencies": {
+    "@nx/devkit": "^17.0.0"
```

Comment:
could nx related code life in the nx plugin?

Reply:
Good question. The code is actually both the ESLint- and Nx-related, so could live in `plugin-eslint`, `nx-plugin` or even `utils`. 

Since we need to release this feature very soon, I wouldn't put it in `nx-plugin` for the time being, simply because unlike `plugin-eslint` it's not ready for release.

I can see us moving this logic out of `plugin-eslint` at some point in the future and probably generalizing it for the CLI as a whole, but I believe it will require some time to figure out how to do that properly. In order to reach our milestone of integrating Code PushUp on our Nx projects ASAP, this is the most straightforward solution I can think of.